src/main/resources/templates/fragments/icons.html

```diff
@@ -8,6 +8,124 @@ <h1>Icons</h1>
+          <path d="M62.1936 26.2492C62.1936 32.2494 57.5573 36.9985 51.9769 36.9985C46.3966 36.9985 41.7603 32.2494 41.7603 26.2492C41.7603 20.2491 46.3966 15.5 51.9769 15.5C57.5573 15.5 62.1936 20.2491 62.1936 26.2492Z" fill="#EEFFE6" stroke="#121111" stroke-width="3"/>
+          <path d="M52.6673 42.4687C50.007 42.5361 46.3167 41.3982 44.7819 40.8794C44.319 40.723 43.818 40.7242 43.3693 40.918C42.0064 41.5066 39.063 42.9074 38.3355 44.3085C37.3801 46.1485 37.38 45.2284 36.4246 49.8279C35.7734 52.9629 36.0099 56.098 36.2266 57.7766C36.3256 58.5441 36.9013 59.1404 37.6595 59.2953C40.159 59.806 46.1378 60.8669 52.6673 60.8667C58.8468 60.8666 64.0395 59.9165 66.4725 59.3819C67.3518 59.1887 67.9524 58.4047 67.9123 57.5053C67.8351 55.7745 67.6167 52.8012 66.9991 49.8279C66.0436 45.2284 66.0436 46.148 65.0882 44.3085C64.4032 42.9895 62.2447 41.6704 61.0777 41.0293C60.5937 40.7634 60.0205 40.7415 59.4989 40.9229C58.0476 41.4277 54.9475 42.4109 52.6673 42.4687Z" fill="#EEFFE6" stroke="#121111" stroke-width="3"/>
+        </svg>
+      </td>
+    </tr>
+    <tr>
+      <td>timeoutClock</td>
+      <td>
+        <svg th:fragment="timeoutClock" width="101" height="75" viewBox="0 0 101 75" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" clip-rule="evenodd" d="M34.2903 66.5636C28.1328 63.5239 21.3046 60.8861 17.736 55.0167C14.0774 48.9991 14.2464 41.5404 14.8045 34.5141C15.3699 27.3947 16.4287 19.9642 20.9587 14.4407C25.4519 8.962 32.5411 6.67831 39.3178 4.62433C45.8713 2.638 52.7714 0.942114 59.3569 2.80237C65.8329 4.63166 70.8052 9.58001 75.1264 14.7393C79.2393 19.6498 81.8154 25.4265 83.488 31.6133C85.2726 38.2142 87.2862 45.1658 85.1057 51.6515C82.8788 58.2755 77.4793 63.3811 71.5602 67.0926C65.854 70.6706 59.1564 72.2377 52.4271 72.1422C45.9701 72.0505 40.0788 69.4211 34.2903 66.5636Z" fill="#9BCBEB"/>
+          <circle cx="51" cy="36.5" r="23" fill="white" stroke="black" stroke-width="3"/>
+          <path d="M36.5 43L23.5 64H49.5L36.5 43Z" fill="#FFC240" stroke="black" stroke-width="2" stroke-linejoin="round"/>
+          <path d="M49.5 25V38.5L59.5 46.5" stroke="black" stroke-width="3" stroke-linecap="round"/>
+          <path d="M36.5 50.4053V57.479" stroke="black" stroke-width="2" stroke-linejoin="round"/>
+          <path d="M36.5 58.6575L36.5 61.0154" stroke="black" stroke-width="2" stroke-linejoin="round"/>
+        </svg>
+
+      </td>
+    </tr>
+    <tr>
+      <td>filledLock</td>
+      <td>
+        <svg th:fragment="filledLock" width="19" height="30" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Some of these icons look like they're from Google Material Icons, which comes in directly from [Honeycrisp](https://honeycrisp.herokuapp.com/cfa/styleguide#atoms-icons).

<img width="559" alt="new-icons" src="https://github.com/codeforamerica/form-flow/assets/9101728/bb5f4045-2c0c-48bc-a076-059896218561">


Some don't have a class, like the `filledLock` or `lock`. But you could use character codepoint, for example:

```
<i class="icon-">&#xe897;</i>
```

Which would give you:
<img width="579" alt="Screenshot 2023-10-26 at 4 14 22 PM" src="https://github.com/codeforamerica/form-flow/assets/9101728/cd0fa779-9d83-46b0-b30a-14ec0630b47d">


The full list of [codepoint reference are here](https://github.com/google/material-design-icons/blob/f5f56570741833bdd36463f1f1b6b7d4edd3f9c1/font/MaterialIconsOutlined-Regular.codepoints), linked from Honeycrisp. 

The format would be `<i class="icon-">&#INSERT_CODEPOINT_HERE;</i>`.

If there's an icon that can't be accessed via codepoint, then I'd be ok with having them here. But we're already loading the icon via a material icon font and this could be a redundant home for them.

Reply:
🤯